hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -2086,6 +2086,8 @@ int vlapic_create(struct vcpu *vcpu)
 				(uint64_t)DEFAULT_APIC_BASE +
```

Comment:
Even at very beginning, it may fail..

int vlapic_create(struct vcpu *vcpu)
{
        void *apic_page = alloc_page();
        struct acrn_vlapic *vlapic = calloc(1U, sizeof(struct acrn_vlapic));


Given that we want to use static memory allocation more, how about we do along this direction?
for now, let us move apic_page into vlapic structure...